modules/compliance-supported-profiles.adoc

```diff
@@ -20,69 +23,81 @@ The Compliance Operator provides the following compliance profiles:
 
-|rhcos4-stig
-|Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) for Red Hat Openshift
-|Node
-|1.3.0+
-|link:https://public.cyber.mil/stigs/downloads/[DISA-STIG] ^[1]^
-|`x86_64`
-|Red Hat OpenShift Service on AWS with hosted control planes (ROSA HCP) - requires 1.5.0+
-
-|ocp4-stig-node
-|Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) for Red Hat Openshift
-|Node
-|1.3.0+
-|link:https://public.cyber.mil/stigs/downloads/[DISA-STIG] ^[1]^
-|`x86_64`
-|Red Hat OpenShift Service on AWS with hosted control planes (ROSA HCP) - requires 1.5.0+
-
-|ocp4-stig
-|Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) for Red Hat Openshift
-|Platform
-|1.3.0+
-|link:https://public.cyber.mil/stigs/downloads/[DISA-STIG] ^[1]^
-|`x86_64`
-|
-
-|ocp4-cis-1-4
-|CIS Red Hat OpenShift Container Platform 4 Benchmark v1.4.0
+|ocp4-cis ^[1]^
+|CIS {product-title} 4 Benchmark v1.5.0
 |Platform
-|1.2.0+
+|1.4.1+
 |link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
 |`x86_64`
  `ppc64le`
  `s390x`
 |
 
-|ocp4-cis-node-1-4
-|CIS Red Hat OpenShift Container Platform 4 Benchmark v1.4.0
-|Node ^[2]^
+|ocp4-cis-1-4 ^[3]^
+|CIS {product-title} 4 Benchmark v1.5.0
+|Platform
 |1.2.0+
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[4]^
 |`x86_64`
  `ppc64le`
  `s390x`
-|Red Hat OpenShift Service on AWS with hosted control planes (ROSA HCP) - requires 1.5.0+
+|
 
-|ocp4-cis
-|CIS Red Hat OpenShift Container Platform 4 Benchmark v1.5.0
+|ocp4-cis-1-5
+|CIS {product-title} 4 Benchmark v1.5.0
 |Platform
-|1.4.1+
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
-|`x86_64`
- `ppc64le`
- `s390x`
 |
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[4]^
+|`x86_64`
+|{product-rosa} with {hcp} (ROSA HCP) - requires 1.5.0+
 
-|ocp4-cis-node
-|CIS Red Hat OpenShift Container Platform 4 Benchmark v1.5.0
+|ocp4-cis-node ^[1]^
+|CIS {product-title} 4 Benchmark v1.5.0
 |Node ^[2]^
 |1.4.1+
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[4]^
 |`x86_64`
  `ppc64le`
  `s390x`
-|Red Hat OpenShift Service on AWS with hosted control planes (ROSA HCP) - requires 1.5.0+
+|{product-rosa} with {hcp} (ROSA HCP) - requires 1.5.0+
+
+|ocp4-cis-node-1-4 ^[3]^
+|CIS {product-title} 4 Benchmark v1.4.0
+|Node ^[2]^
+|1.2.0+
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[4]^
+|`x86_64`
+|{product-rosa} with {hcp} (ROSA HCP) - requires 1.5.0+
+
+|ocp4-cis-node-1-5
+|CIS {product-title} 4 Benchmark v1.5.0
+|Node ^[2]^
+|1.2.0+
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[4]^
+|`x86_64`
+|{product-rosa} with {hcp} (ROSA HCP) - requires 1.5.0+
```

Comment:
I thought this was supported on Power too? @AniruddhaNayek do we test v1.5?  The platform column is a bit confusing (maybe more context?